src/components/Grid/Column.js

```diff
@@ -22,9 +22,12 @@ export const Column = React.memo((props) => {
    * tabIndex and point columnRef to itself so it gets the focus.
    */
   const interactiveRender = (props) => {
+    const klasses = props.header
```

Comment:
I hope klasses is not typo . i like the variable name

Reply:
`klass` with a `k` is generally used instead of `class` to prevent reserved keywords from being used and to indicate that the variable is a class.  (https://stackoverflow.com/questions/4299289/what-is-the-difference-between-class-and-klass-in-ruby).